pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -112,7 +76,8 @@ type ServiceBindingRequest struct {
 type ServiceBindingRequestList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
-	Items           []ServiceBindingRequest `json:"items"`
+
+	Items []ServiceBindingRequest `json:"items"`
```

Comment:
What can we do on our vscode to ensure it doesn't change again? :)

Reply:
@sbose78 I'm afraid because I added a new line, to follow the same standard than on Spec field, the Go-Format took care of setting up the code again. So as long as we don't change the surroundings anymore, that should not change. :-)